manifests/instance/config.pp

```diff
@@ -210,6 +211,9 @@
   # Configure Init script
   #
   if $::osfamily == 'RedHat' and $::tomcat::params::systemd {
+    if !empty($setenv) {
+      warning "\$setenv is deprecated (current value: ${setenv}, please use \$java_opts instead"
```

Comment:
Is `$setenv` still working ? If not, we should `fail` here or we'll break existing configs.